jdaviz/core/template_mixin.py

```diff
@@ -3456,8 +3456,6 @@ def __init__(self, *args, **kwargs):
                                       manual_options=['None', 'Surrounding'],
                                       default_mode='first',
                                       filters=['is_spectral'])
-        self.app.hub.subscribe(self, ViewerVisibleLayersChangedMessage,
-                               lambda _: self._clear_cache('continuum_marks'))
```

Comment:
Did some other added logic make this redundant? (Trying to think of how best to test that this doesn't introduce issues - I guess this would be when hiding the spectrum that is currently selected?)

Reply:
That currently doesn't work on main without this removed, but it makes sense to me that toggling data visibility should toggle associated marks visibility. I can make a small follow up ticket